src/versionbits.h

```diff
@@ -64,8 +64,10 @@ class AbstractThresholdConditionChecker {
     virtual int Threshold(const Consensus::Params& params) const =0;
 
 public:
-    /** Returns the numerical statistics of an in-progress BIP9 softfork in the current period */
-    BIP9Stats GetStateStatisticsFor(const CBlockIndex* pindex, const Consensus::Params& params) const;
+    /** Returns the numerical statistics of an in-progress BIP9 softfork in the period including pindex
+     * If provided, signals is set to true/false based on whether each block in the period signalled
+     */
+    BIP9Stats GetStateStatisticsFor(const CBlockIndex* pindex, const Consensus::Params& params, std::vector<bool>* signals = nullptr) const;
```

Comment:
`signals` is not a particularly good name for this parameter, imo, i had no idea what was meant (or what the index is) at first, maybe `signallingBlocks`?

Reply:
Changed to `signalling_blocks`